magefiles/airflow.go

```diff
@@ -39,7 +39,13 @@ func Airflow(arg string) error {
 func startAirflow() error {
 	fmt.Println("Starting airflow...")
 
-	err := sh.Run("mkdir", "-p", "developer/airflow/opt")
+	// Remove developer/airflow/opt
+	err := sh.Run("rm", "-rf", "developer/airflow/opt/")
```

Comment:
I found this to be helpful. Without it, I got problems with nested copies when repeatedly trying to run airflow?

Reply:
Nested copies seems strange. I don't think I had that behavior. However, I don't think this is a problem.